queries/c_sharp/highlights.scm

```diff
@@ -305,10 +305,10 @@
  "}"
  "("
  ")"
- "<"
- ">"
```

Comment:
Restore these, > and < are also operators

Reply:
That's the `@punctuation.bracket` list; `<` and `>` are still in the `@operator` list (see line 270 and 272). These characters being in this list unconditionally is the problem

Reply:
Whoops, yeah